index.rst

```diff
@@ -165,12 +165,4 @@ Contains detailed instructions on how to set the required environment for operat
 System Testing, Deployments, and Upgrades
 ==================================================================
```

Comment:
You might get a warning since there are more = signs than characters in the title.

Reply:
yeah.. need to make a pass in the whole module.. but I don't want to make changes to the content when moving this much files around..